examples/seismic/model.py

```diff
@@ -614,18 +654,33 @@ def vp(self, vp):
 
 
 class ModelElastic(GenericModel):
-    """The physical model used in seismic inversion processes.
-    :param origin: Origin of the model in m as a tuple in (x,y,z) order
-    :param spacing: Grid size in m as a Tuple in (x,y,z) order
-    :param shape: Number of grid points size in (x,y,z) order
-    :param space_order: Order of the spatial stencil discretisation
-    :param vp: P-wave velocity in km/s
-    :param vs: S-wave velocity in km/s
-    :param nbpml: The number of PML layers for boundary damping
-    :param rho: Density in kg/cm^3 (rho=1 for water)
-    The :class:`ModelElastic` provides a symbolic data objects for the
+    """
+    The physical model used in seismic inversion processes.
+
+    Parameters
+    ----------
+    origin :
+        Origin of the model in m as a tuple in (x,y,z) order.
+    spacing :
+        Grid size in m as a Tuple in (x,y,z) order.
+    shape : int
+        Number of grid points size in (x,y,z) order.
+    space_order : int
+        Order of the spatial stencil discretisation.
+    vp : float
+        P-wave velocity in km/s.
+    vs : float
+        S-wave velocity in km/s.
```

Comment:
or Array for vp/vs, origin/spacing,... as above for other Model.

Reply:
'array' should be 'array-like' (or 'array_like') everywhere as it's the numpy convention to refer to numpy arrays

Reply:
definitely not "Array" with the capitol A

Reply:
fixed